datavault-broker/src/main/resources/release-sql/3.2.0/role-assignments-index-update.sql

```diff
@@ -0,0 +1,11 @@
+ALTER TABLE Role_assignments DROP CONSTRAINT `FK_2j93l0yx740l9r9hjeinwia1f`;
```

Comment:
Are these names actually predictable?

Reply:
Yeah. They really don't look like they would be but they came out the same when the UoE guys generated the creation script on their end as I had in my local db

Reply:
Huh, never something I'd thought about. TIL.